code/game/machinery/doors/firedoor.dm

```diff
@@ -44,19 +44,25 @@
 	else
 		. += span_notice("The bolt locks have been <i>unscrewed</i>, but the bolts themselves are still <b>wrenched</b> to the floor.")
 
+/obj/machinery/door/firedoor/proc/CalculateAffectingAreas() //SKYRAT EDIT CHANGE
```

Comment:
Couldn't this be put in modular? We already have a firedoor.dm in master files

Reply:
Wait for the updates